runtime/errors.go

```diff
@@ -39,10 +39,15 @@ type Error struct {
 }
 
 func newError(err error, context Context) Error {
+	codes := make(map[common.Location]string, len(context.codes))
+	for location, code := range context.codes {
```

Comment:
I'm not really sure why ranging over a map is a lint error in cadence.  Could you give me a quick explanation?